geps/gep-713.md

```diff
@@ -1338,8 +1338,8 @@ If a `sectionName` is specified, but does not exist on the targeted object, the
 and the policy implementation should record a `resolvedRefs` or similar Condition in the Policy's status.
 
 When multiple Policies of the same type target the same object, one with a `sectionName` specified, and one without,
-the one with a `sectionName` is more specific, and so will have all its settings apply. The less-specific Policy will
-not attach to the target.
+the one with a `sectionName` is more specific, and so will have all its settings applied to the named target. The less-specific Policy will
+still be applied but MUST not affect the named target.
```

Comment:
```suggestion
still be applied but MUST not affect the named target i.e. it will be applied to all other sections which have not been targeted by another Policy 
```